doc/nrf/protocols/matter/getting_started/adding_clusters.rst

```diff
@@ -181,50 +181,23 @@ This is needed to properly model the sensor's behavior.
 The :file:`src/app_task.cpp` file contains the main loop of the application.
 Complete the steps in the following subsections to modify the main loop.
 
-Edit the event queue
-====================
-
-The main application loop is based on an ``AppEventQueue``, on which events are posted by ZCL callbacks and by other application components, such as Zephyr timers.
-In each iteration, an event is dequeued and a corresponding event handler is called.
-
 Add new events
---------------
-
-After you copied the template sample application, the following events are used in the application (:file:`src/app_event.h`):
-
-* :c:enum:`Button` - For general operation of **Button 1**.
-* :c:enum:`ButtonPushed` - For pressing **Button 1**.
-* :c:enum:`ButtonReleased` - For releasing **Button 1**.
-* :c:enum:`Timer` - For factory reset timeout.
-* :c:enum:`UpdateLedState` - For updating of the **LED 1** state.
+==============
 
-To model the behavior of the sensor, add the following new :c:enum:`AppEventType` events:
-
-* :c:enum:`SensorActivate` - For sensor activation.
-* :c:enum:`SensorDeactivate` - For sensor deactivation.
-* :c:enum:`SensorMeasure` - For sensor measurement update.
-
-For example, the edited :c:enum:`AppEventType` can look as follows:
+The main application uses a task queue managed by the ``task_executor`` common module, on which events are posted by ZCL callbacks and by other application components, such as Zephyr timers.
```

Comment:
```suggestion
The main application uses a task queue managed by the ``task_executor`` common module, on which tasks are posted by ZCL callbacks and by other application components, such as Zephyr timers.
```
Maybe it's better not to use `task` and `event` interchangeably? It may not be clear that you mean the same thing by both terms. The common module only defines `task` and `event` is now a concept that an application may or may not use.

Reply:
Right, will change all occurrences to task.